types/index.d.ts

```diff
@@ -0,0 +1,75 @@
+// Type definitions for @nuxtjs/auth 4.8
+// Project: https://auth.nuxtjs.org
+// TypeScript Version: 3.1
+
+export interface Storage {
+  setUniversal(key: string, value: any, isJson?: boolean): string;
+  getUniversal(key: string, isJson?: boolean): any;
+  syncUniversal(key: string, defaultValue: any, isJson?: boolean): any;
+  // Local State
+  setState(key: string, val: any): string;
+  getState(key: string): string;
+  watchState(key: string, handler: (newValue: any) => void): any;
+  // Cookies
+  setCookie(key: string, val: any, options?: object): any;
+  getCookie(key: string, isJson?: boolean): any;
+  // Local Storage
+  setLocalStorage(key: string, val: any, isJson?: boolean): any;
+  getLocalStorage(key: string, isJson?: boolean): any;
+}
+
+export interface Auth<T = any> {
+  ctx: any;
+  $state: any;
+  $storage: Storage;
+  user: Partial<T>;
+  loggedIn: boolean;
+  loginWith(strategyName: string, ...args: any): Promise<never>;
+  login(...args: any): Promise<never>;
+  logout(): Promise<never>;
+  fetchUser(): Promise<never>;
+  fetchUserOnce(): Promise<never>;
+  hasScope(scopeName: string): boolean;
+  setToken(strategyName: string, token?: string): string;
```

Comment:
`setToken(strategyName, token)`, `getToken(strategyName)` and `syncToken(strategyName)` has been deprecated. They were moved to Token class and refactored. You can use them like this `$auth.token.set(token)`, `$auth.token.get()` and `$auth.token.sync()`.

Reply:
lol. I guess that's what happens when a PR sits for a while :/ I'll work on this. Thanks for the comments!!